cmd/torch.js

```diff
@@ -134,6 +135,79 @@ function outputDTraceText(stacks) {
     adaptor.resume();
 }
 
+function outputSVG(stacks) {
+    var path = require('path');
+    var os = require('os');
+    var fs = require('fs');
+    var exec = require('child_process').exec;
+
+    var adaptor = new StackVisAdaptor(stacks);
+    var tempFile = path.join(os.tmpDir(), 'stack.raw');
+    var tempFile2 = path.join(os.tmpDir(), 'stack.folded');
+
+    adaptor.pipe(fs.createWriteStream(tempFile))
+        .once('finish', fileWritten);
+
+    adaptor.resume();
+
+    function fileWritten() {
+        var stackCollapse = path.join(
+            __dirname,
+            '..',
+            'bg-flamegraphs',
+            'stackcollapse-stap.pl'
+        );
+
+        var command = [
+            'perl ',
+            stackCollapse,
+            ' ',
+            tempFile,
+            ' | tr -d "\\0"'
```

Comment:
at least add a note that this is a hack around broken output from this same project, and should be removed once said bug is fixed.